src/upload/catalog/controller/extension/payment/libwebpay_rest/HealthCheck.php

```diff
@@ -220,8 +177,7 @@ public function setInitTransaction(){
         $buyOrder = "_Healthcheck_";
         $sessionId = uniqid();
         $returnUrl = "https://webpay3gint.transbank.cl/filtroUnificado/initTransaction";
-        $finalUrl = "https://webpay3gint.transbank.cl/filtroUnificado/initTransaction";
-        $result = $transbankSdkWebpay->initTransaction($amount, $sessionId, $buyOrder, $returnUrl, $finalUrl);
+        $result = $transbankSdkWebpay->initTransaction($amount, $sessionId, $buyOrder, $returnUrl);
```

Comment:
Should we keep the "initTransaction" naming? in REST the operation is called `create`

Reply:
cc @a1b4 what do you think?

Reply:
I agree with @eugenioLopezRamos we need to change the name to create in the Rest case.

Reply:
Ok, changing.